resources/benchmark-runner.mjs

```diff
@@ -327,6 +327,10 @@ export class BenchmarkRunner {
         let asyncStartTime;
         let asyncTime;
         const runSync = () => {
+            if (params.warmupBeforeSync) {
+                const startTime = performance.now();
+                while (performance.now() - startTime < 100); // Infinite loop for 100ms.
```

Comment:
Could we make this configurable and use an int for `params.warmupBeforeSync` instead?